src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -161,16 +154,13 @@ private function decideConsensus(TokenInterface $token, array $attributes, $obje
      */
     private function decideUnanimous(TokenInterface $token, array $attributes, $object = null)
     {
-        $grant = 0;
         foreach ($attributes as $attribute) {
             foreach ($this->voters as $voter) {
                 $result = $voter->vote($token, $object, array($attribute));
 
                 switch ($result) {
                     case VoterInterface::ACCESS_GRANTED:
-                        ++$grant;
-
-                        break;
+                        return true;
```

Comment:
I think those changes are wrong for the `unanimous` strategy.

> `unanimous` only **grant** access **if none** of the voters **has denied access**;

That does not mean one `ACCESS_GRANTED` is enough. You still need to iterate over every other voter to check none of them denied access.